redfishAPI.cfg

```diff
@@ -17,5 +17,10 @@
     "trace": "4",
     "urltimeout": "30",
     "usefinalslash": "True",
-    "username": ""
+    "username": "",
+    "listenerusessl": "False",
+    "listenerip": "localhost",
+    "listenerport": "8080",
+    "certfile": "cert.pem",
+    "keyfile": "server.key"
```

Comment:
The certificate PEM file and private key PEM file should not be hard-coded in the configuration file. The user should specify the security files required.

Reply:
Done.